frontend/vue.config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    configureWebpack: {
+        devServer: {
+            proxy: {
+                '/vehicles|/routes|/adminMessage|/updates|/stops': {
+                    target: 'https://shuttles.rpi.edu',
```

Comment:
What's the recommended way to configure this to point at a local instance? Mostly for people working on the backend